src/renderer/components/+catalog/material-tooltip/material-tooltip.tsx

```diff
@@ -0,0 +1,28 @@
+import React from "react";
+import { makeStyles, Tooltip, TooltipProps } from "@material-ui/core";
+
+const useStyles = makeStyles(() => ({
+  arrow: {
+    color: "#18191c",
```

Comment:
Light theme coloring would come with another PR

Reply:
Wouldn't it make sense to have it in this PR though?

Reply:
Cache issue? What is wrong with just putting these into the default MuiProvider which is called in `bootstrap.tsx`? That way we don't even need this special component.

Reply:
Recently we removed global styles pointing MuiProvider to allow redesign tooltips from different places. Meanwhile, the cache bug is fixed with `make dev`, so css variables are set here. So this issue is fixed and no longer relevant.